.storybook/preview.tsx

```diff
@@ -3,36 +3,45 @@ import '../src/index.scss'
 import { SeamProvider } from '@seamapi/react'
 import type { Preview } from '@storybook/react'
 
-import { fakePublishableKey, fakeUserIdentifierKey } from './seed-fake.js'
-
-const isProd = process.env['NODE_ENV'] === 'production'
-const useFake = process.env['STORYBOOK_SEAM_ENDPOINT'] == null
+const useFake =
+  ['1', 'true'].includes(
+    process.env['STORYBOOK_DISABLE_FAKE']?.toLowerCase()
```

Comment:
Ah, my suggestion broke the types. Here is one fix
```suggestion
  ['1', 'true'].includes(
    process.env['STORYBOOK_DISABLE_FAKE']?.toLowerCase() ?? ''
```